Sources/StreamChat/Models/Attachments/ChatMessageAttachmentEnvelope.swift

```diff
@@ -23,12 +23,12 @@ public extension ChatMessageAttachmentEnvelope {
         )
     }
 
-    init(localFileURL: URL) throws {
+    init(localFileURL: URL, attachmentType: AttachmentType) throws {
```

Comment:
@skorepak do we really need this? 🤔 This API allows to give a custom attachment type but the attachment can't be updated when the file is uploaded since the `type` is different from `file/image`

Reply:
basically wondering what the use case is 🙂 Sending an image as a file attachment or ..? If we are sure the type has to be passed explicitly other parts of attachment uploading flow (`AttachmentDTO.update(uploadedFileURL:)`) should be updated accordingly

Reply:
We need this to distinguish between the image uploaded from file picker vs from image picker. Without this, when the image was picked through a file picker, we showed the image as an image, even tho it was picked as a file.